python/dgl/graphbolt/dataset.py

```diff
@@ -46,6 +46,11 @@ def test_set(self) -> ItemSet or ItemSetDict:
         """Return the test set."""
         raise NotImplementedError
 
+    @property
```

Comment:
You might not need update this dataset, but just allow ItemSet initiate by integer.
https://github.com/dmlc/dgl/blob/master/python/dgl/graphbolt/itemset.py

Reply:
Should I allow `items` to be integer? Or add another parameters in `__init__`?